packages/lit/src/portal.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  html,
+  LitElement,
+  render,
+  type RenderOptions,
+  type TemplateResult,
+} from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+/**
+ * Renders a template into a portal. Defaults to `document.body`.
+ *
+ * Note that every time the parent component re-renders, the portal will be re-called.
+ *
+ * See https://lit.dev/docs/components/rendering/#writing-a-good-render()-method
+ *
+ * @example
+ * ```ts
+ * render() {
+ *   return html`${showPortal
+ *     ? html`<affine-portal .template=${portalTemplate}></affine-portal>`
+ *     : null}`;
+ * };
+ * ```
+ */
+@customElement('affine-portal')
```

Comment:
Maybe `blocksuite-portal` ?
Should we keep affine as prefix for our common elements? @doodlewind

Reply:
Yes should be `blocksuite-portal`. Having this prefix sounds like a good convention to follow, when providing components that are expected to be used outside of the editor.